Tensile/TensileCreateLibrary.py

```diff
@@ -1373,36 +1372,6 @@ def writeBenchmarkClientFiles(libraryWorkingPath, tensileSourcePath, solutions,
     return (codeObjectFiles, newLibrary)
 
 
-def WriteClientLibraryFromSolutions(solutionList, libraryWorkingPath, tensileSourcePath=None):
-
-    if tensileSourcePath == None:
-        tensileSourcePath = os.path.dirname(os.path.realpath(__file__))
-    firstSolution = deepcopy(solutionList[0])
-    problemType = firstSolution["ProblemType"].state
-    problemType["DataType"] = problemType["DataType"].value
-    problemType["DataTypeA"] = problemType["DataTypeA"].value
-    problemType["DataTypeB"] = problemType["DataTypeB"].value
-    problemType["DestDataType"] = problemType["DestDataType"].value
-    problemType["ComputeDataType"] = problemType["ComputeDataType"].value
-    problemType["MathDataTypeA"] = problemType["MathDataTypeA"].value
-    problemType["MathDataTypeB"] = problemType["MathDataTypeB"].value
-    problemType["F32XdlMathOp"] = problemType["F32XdlMathOp"].value
-    cxxCompiler = globalParameters["CxxCompiler"]
-
-    effectiveWorkingPath = os.path.join(libraryWorkingPath, "library")
-    ensurePath(effectiveWorkingPath)
-    mataDataFilePath = os.path.join(effectiveWorkingPath, "metadata.yaml")
-
-    metaData = {"ProblemType": problemType}
-    LibraryIO.writeYAML(mataDataFilePath, metaData)
-
-    codeObjectFiles, newLibrary = writeBenchmarkClientFiles(
-        libraryWorkingPath, tensileSourcePath, solutionList, cxxCompiler
-    )
-
-    return (codeObjectFiles, newLibrary)
-
-
```

Comment:
Why is this deleted? No longer needed?

Reply:
I was planning to remove the deepcopy but decided to look to see if this function was used anywhere and I cannot find evidence of it being used.